src/test/java/co/rsk/federate/bitcoin/BtcTransactionBuilder.java

```diff
@@ -0,0 +1,105 @@
+package co.rsk.federate.bitcoin;
+
+import co.rsk.bitcoinj.core.Address;
+import co.rsk.bitcoinj.core.BtcTransaction;
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.bitcoinj.core.NetworkParameters;
+import co.rsk.bitcoinj.core.TransactionInput;
+import co.rsk.bitcoinj.core.TransactionOutPoint;
+import co.rsk.bitcoinj.core.TransactionOutput;
+import co.rsk.bitcoinj.core.TransactionWitness;
+import co.rsk.bitcoinj.script.Script;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public final class BtcTransactionBuilder {
+
+    private final NetworkParameters networkParameters;
+    private final Map<Integer, TransactionWitness> transactionWitnesses = new HashMap<>();
+    private final List<TransactionInput> inputs = new ArrayList<>();
+    private final List<TransactionOutput> outputs = new ArrayList<>();
+
+    public BtcTransactionBuilder(NetworkParameters networkParameters) {
+        this.networkParameters = networkParameters;
+    }
+
+    public BtcTransactionBuilder withInput(TransactionInput transactionInput) {
+        inputs.add(transactionInput);
+        return this;
+    }
+
+    public BtcTransactionBuilder withInput(TransactionOutput transactionOutput) {
+        TransactionInput transactionInput = new TransactionInput(networkParameters, null,
```

Comment:
you could use the `InputBuilder` here right?

Reply:
Done